docs/resources/tenant.md

```diff
@@ -377,7 +377,7 @@ resource "fusionauth_tenant" "example" {
     modify_encryption_scheme_on_login = false
   }
   password_validation_rules {
-    max_length = 256
+    max_length = 50
```

Comment:
Is this trying to document the FusionAuth default configuration? If so, the actual defaults are min `8` and max `256`. 
Or is this just a recommended default for the terraform users?

Reply:
I set this back to 256, since it is valid with this scheme. I also updated the docs to include the new `max_length` constraint when using `bcrypt`.